src/common.js

```diff
@@ -4,11 +4,23 @@
 
 var os = require('os');
 var fs = require('fs');
+var path = require('path');
 var glob = require('glob');
 var shell = require('..');
 
 var DEFAULT_ERROR_CODE = 1;
 var shellMethods = Object.create(shell);
+var PATH = 'PATH';
+
+// windows calls it's path 'Path' usually, but this is not guaranteed.
```

Comment:
I posted a follow up comment. It looks to me like they're making a PR to add this to the documentation (I didn't read the PR yet, so I could be mistaken). This seems to indicate that this is the intended behavior on Node on Windows.

My *guess* is that `process.env` has always had case-insensitive keys on Windows, but it was just never documented. I think it's ok to rely on that (unless we hear otherwise).